modules/microshift-mirroring-prereqs.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mirroring-prereqs_{context}"]
+= Mirroring prerequisites
+
+Follow the instructions in the "Configuring credentials that allow images to be mirrored" link provided in the Additional resources to create a `~/.pull-secret-mirror.json` file including the user credentials for accessing the mirror.
```

Comment:
The document I am reading suggests that you edit .pull-secret.json not create a file called .pull-secret-mirror.json

Am I following the wrong document ?
https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-installation-images.html#installation-adding-registry-pull-secret_installing-mirroring-installation-images

Reply:
Same as above.

Reply:
Step 2 in the procedure is creating a new file, which is then edited by appending the mirror-specific info.